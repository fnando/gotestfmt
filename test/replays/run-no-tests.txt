{"Time":"2023-11-03T09:24:33.675428-07:00","Action":"start","Package":"github.com/fnando/bolt/test/reference/no-tests"}
{"Time":"2023-11-03T09:24:33.675478-07:00","Action":"output","Package":"github.com/fnando/bolt/test/reference/no-tests","Output":"?   \tgithub.com/fnando/bolt/test/reference/no-tests\t[no test files]\n"}
{"Time":"2023-11-03T09:24:33.675485-07:00","Action":"skip","Package":"github.com/fnando/bolt/test/reference/no-tests","Elapsed":0}
