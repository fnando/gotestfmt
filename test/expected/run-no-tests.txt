

Finished in 0s, 0 tests, 0 failures, 0 skips, 0 benchmarks

Coverage:

[0.0%] github.com/fnando/bolt/test/reference/no-tests
